src/amo/components/CollectionManager/index.js

```diff
@@ -189,9 +212,11 @@ export const extractId = (ownProps: Props) => {
   return `collection-${collection ? collection.slug : ''}`;
 };
 
-export const mapStateToProps = (state: {| api: ApiStateType |}) => {
+export const mapStateToProps = (state: {| api: ApiStateType, users: UsersStateType, |}) => {
+  const currentUser = getCurrentUser(state.users);
   return {
     siteLang: state.api.lang,
+    currentUsername: currentUser && currentUser.username,
```

Comment:
All we need is the current user's username, in the case of `create`, so this seemed like a reasonable way to get it. I suppose another option would have been to pass the username in as a property from the caller. I'm not sure which is best in this situation.